components/eventing-controller/controllers/subscription/nats/reconciler_test.go

```diff
@@ -254,6 +255,83 @@ func testCleanEventTypes(id int, eventTypePrefix, natsSubjectToPublish, eventTyp
 	})
 }
 
+// testUpdateSubscriptionStatus tests if the reconciler can create and update the Status of a Subscription as expected.
+func testUpdateSubscriptionStatus(id int, eventTypePrefix, natsSubjectToPublish, eventTypeToSubscribe string) bool {
+	return When("updating the clean event types in the Subscription status", func() {
+		It("should mark the Subscription as ready", func() {
+			//  set default expectations
+			defaultCondition := eventingv1alpha1.MakeCondition(
+				eventingv1alpha1.ConditionSubscriptionActive,
+				eventingv1alpha1.ConditionReasonNATSSubscriptionActive,
+				v1.ConditionTrue, "")
```

Comment:
Why is the message empty ?

Reply:
cause for "true" no message was used till now (see my other comment)